### ISSUE TITLE
No Gateway or Ingress for OPEA pipeline

### ISSUE THREAD

**Issue by Jiangbo-W** [opea-project/GenAIInfra#197]
Gateway/Ingress is the key feature for users to manage the incoming traffic and access for the services.

**Comment by KfreeZ** [comment]
yes, agree that gateway is key for OPEA, we're working on it, will submit a RFC soon.

**Comment by endomorphosis** [comment]
I used openvpn for this and for connecting machines across regions, I had to do some custom work to get this working, advise me if you need the .yaml files for this.

**Comment by daixiang0** [comment]
Here is the RFC.

**Comment by jfding** [comment]
Let close GMC related issues at first, but for possible future new requests we can reopen it.
